dataset/aoi.py

```diff
@@ -91,6 +91,8 @@ def asset_by_common_name(self) -> Dict:
         if self._assets_by_common_name is None:
             self._assets_by_common_name = OrderedDict()
             for name, a_meta in self.item.assets.items():
+                if name == 'FOOTPRINT':
+                    self._assets_by_common_name['FOOTPRINT'] = {'href': a_meta.href, 'name': name}
```

Comment:
Do you transfer what I did in `inference/SingleBandItemEO.py`  and `inference/InferenceDataset.py` ?